node/node.go

```diff
@@ -147,6 +157,33 @@ func NewNode(ctx context.Context, conf config.NodeConfig, nodeKey crypto.PrivKey
 	return node, nil
 }
 
+func (n *Node) InitGenesisChunks(chunkSize int64) error {
```

Comment:
Tendermint does not allow for this function to take in a chunk size: https://github.com/tendermint/tendermint/blob/master/internal/rpc/core/env.go#L152

I only did so in order to make writing the test a bit easier. The default chunk Size is quite big and cumbersome to test for if set to it's default: https://github.com/celestiaorg/optimint/pull/287/files#diff-375d57e386f20eaa5f09f02bb9d28bfc48ac3dca18d0325f59492208219e5618R45

Happy to change or hear thoughts

Reply:
Wait, this `chunkSize` isn't actually used in the function body...

Reply:
Ah you are right....thank you for catching that. I think I may have got a bit into a rabbit hole with the testing and thought my issue stemmed from the chunk size. Have updated to remove the `chunkSize` param now